project/src/com/google/daggerqueryui/page-style.css

```diff
@@ -15,3 +15,13 @@
 #query-text-field {
     margin-top: 5%;
 }
+
+.interactive-input-container .query-name {
+    position: absolute;
+    width: auto;
+    display: none;
+    border: 0px;
+    color: #09AF00;
+    background: #FFF !important;
```

Comment:
nit: I think `white` is more readable than `#FFF`, but it may be a matter of taste.

Reply:
At first I tried different hex values for the color, but then I decided to stick with white - let's replace the hex name with `white`, it really looks better.